src/ioc_scan/ioc_scanner.py

```diff
@@ -158,4 +158,20 @@ def main(blob=None, root="/"):
 
 
 if __name__ == "__main__":
-    sys.exit(main())
+    import argparse
+
+    parser = argparse.ArgumentParser(
+        description="Indicators of compromise (IoC) scanning tool."
+    )
+    parser.add_argument(
+        "-f", "--file", dest="hashfile", help="Get IOC hashes from specified file."
+    )
+    args = parser.parse_args()
+
+    if args.hashes_file:
```

Comment:
Shouldn't this be `if args.hashfile:`?

Reply:
Ah, glad you caught that mistake. I used that name when originally creating the parser, but realized it didn't align with how things are named in `ioc_scan_cli.py`. I missed that one reference when changing the name to be consistent, but it should be fixed in 6058935.